tests/js/client/resilience/move/moving-shards-cluster.inc

```diff
@@ -34,23 +34,31 @@ const db = arangodb.db;
 const _ = require("lodash");
 const internal = require("internal");
 const {errors, wait} = internal;
-const {debugSetFailAt, debugCanUseFailAt, debugRemoveFailAt, debugClearFailAt} = require('@arangodb/test-helper');
 const supervisionState = require("@arangodb/testutils/cluster-test-helper").supervisionState;
 const queryAgencyJob = require("@arangodb/testutils/cluster-test-helper").queryAgencyJob;
-const {getServersByType, deriveTestSuite, getEndpointById, getUrlById} = require('@arangodb/test-helper-common');
+const {getServersByType, deriveTestSuite, getUrlById} = require('@arangodb/test-helper-common');
 const request = require('@arangodb/request');
 const helper = require("@arangodb/testutils/replicated-logs-helper");
 const lpreds = require("@arangodb/testutils/replicated-logs-predicates");
+const {
+  agency,
+  debugSetFailAt, 
+  arangoClusterInfoGetCollectionInfo,
+  arangoClusterInfoGetCollectionInfoCurrent,
+  debugClearFailAt, 
+  arangoClusterInfoFlush,
+  getDBServers,
+  getEndpointById,
+  getCoordinators
+} = require('@arangodb/test-helper');
 
 // in the `useData` case, use this many documents:
 const numDocuments = 1000;
 
-const dbservers = (function () {
-  return getServersByType('dbserver').map((x) => x.id);
-}());
+const dbservers = getDBServers().map(arangod => arangod.id);
 
 function delaySupervisionFailoverActions(value) {
-  let agents = global.instanceManager.arangods.filter(
+  let agents = getDBServers().filter(
     arangod => arangod.instanceRole === "agent").map(
```

Comment:
This looks wrong. We are calling `getDBServer()` and then type filter on the `instanceRole` being `agent`.